perf-tool/src/agent.cpp

```diff
@@ -137,21 +160,25 @@ JNIEXPORT jint JNICALL Agent_OnAttach(JavaVM* vm, char *options, void *reserved)
     /* If a previous attach operation loaded the library and created
      * the server, don't try to do it again
      */
+   
     if (!server)
     {
         jvmtiError error;
-        printf("On attach initiated with options: %s\n", options);
         parseAgentOptions(options);
-        
+        if (verbose >= 2)
+        printf("On attach initiated with options: %s\n", options);
```

Comment:
Please indent the statement after the `if`. Similar comment for other places in the code